lib/common.sh

```diff
@@ -213,6 +229,21 @@ resolve_ipv4()
         return 1
     fi
 
+    #
+    # Check if the IP-FQDN pair is present in /etc/hosts
+    # 
+    if is_present_in_etc_hosts "$ipv4_addr" "$hname"; then
+        if [ "$fail_if_present_in_etc_hosts" == "true" ]; then
+            eecho "[ERROR] Detected entry $ipv4_addr $hname in /etc/hosts."
+            eecho "$hname is resolving into $ipv4_addr from /etc/hosts."
+            eecho "[Action Required]: Remove the entry for "$hname" in /etc/hosts to ensure proper handling of endpoint IP address changes by AZNFS."
+            return 1
+        else
+            wecho "[WARNING] Detected entry $ipv4_addr $hname in /etc/hosts." 1>/dev/null
+            wecho "[Action Required]: Remove the entry for "$hname" in /etc/hosts to ensure proper handling of endpoint IP address changes by AZNFS." 1>/dev/null
```

Comment:
We have not used [Error] and other such things {except FATAL} in our code anywhere. Lets be consistent. I already have a workitem for such improvements in log. Lets do it for the whole code separately if we get time later.

Reply:
yes, makes sense.